source/develop/release-management/features/network/ipv6_subnets.html.md

```diff
@@ -56,6 +60,40 @@ The following configurations, which are explained in the [IPAM types](#ipam-type
 
 * static addresses
 * stateful DHCPv6
+* stateless DHCPv6
+
+### Why OVN logical routers are required
+
```

Comment:
Default gateway and MTU was missing, too.

Reply:
Done.